pkg/skaffold/sync/sync.go

```diff
@@ -138,6 +139,19 @@ func inferredSyncItem(a *latest.Artifact, e filemon.Events, builds []build.Artif
 	return &Item{Image: tag, Copy: toCopy}, nil
 }
 
+func inferredSyncMap(a *latest.Artifact, insecureRegistries map[string]bool) (map[string][]string, error) {
```

Comment:
This name, `InferredSyncMap()`, seems off as `docker.SyncMap()` returns the tracked add+copies derived from the `Dockerfile`, and it isn't limited to the files in the `inferred:` block.